saml/serviceProviders/aws/provider.go

```diff
@@ -89,6 +90,37 @@ func (a AwsServiceProvider) GetCredentials(saml string, desiredRole string) *sts
 	return out.Credentials
 }
 
+func (a AwsServiceProvider) AssumeRole(creds sts.Credentials, targetRole string) (*sts.Credentials, error) {
+	awsSession, err := session.NewSessionWithOptions(session.Options{
+		Config: aws.Config{Credentials: credentials.NewStaticCredentialsFromCreds(credentials.Value{
+			AccessKeyID:     aws.StringValue(creds.AccessKeyId),
+			SecretAccessKey: aws.StringValue(creds.SecretAccessKey),
+			SessionToken:    aws.StringValue(creds.SessionToken),
+		})},
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	stsClient := sts.New(awsSession)
+	input := &sts.AssumeRoleInput{
+		RoleArn: aws.String(targetRole),
+		RoleSessionName: aws.String("cstavro-test"),
```

Comment:
This'll need to get swapped out to something less testy, but I don't think we need anything complicated to start with. The current model uses the [PID+username](https://github.com/rtkwlf/awn/blob/master/pkg/rtkokta/rtkokta/awsokta.py#L163)

Reply:
hmm, what username are you thinking here? There's no access to a username anywhere here. The default ~/.aws/config that we generate includes a 'role_session_name' that uses email address. Should we make this part of the .bmx config? Maybe default it to the USER env var? That seems to be my macos username. On windows it's USERNAME, should be easy enough to account for both.

Reply:
I think just using the okta username would be sufficient, and we should already have it in the context from `printOptions.User` to be passed to AssumeRole as the 'roleSessionName' context.